main.go

```diff
@@ -143,6 +143,16 @@ func main() {
 				eval = evaluator.New()
 				prev.eval = eval
 
+			case ":setcookie":
```

Comment:
not sure if a command is really necessary here. Telling the user to put their token in `AOC_TOKEN` or `token.txt` seems good enough.

Maybe later we can try to automatically load the cookie from the browser, like some other AoC helpers do.

Reply:
No problem, let me remove it then, just a moment